CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to add as much Loacker to the project as possible.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Adding a PR to add loacker.
+* Being respectful of Ilan's insatiable hunger for the wafer.
+* Accepting Ilan's devouring of hills of loacker wafers.
+* Focusing on what is best for the community.
+* Showing empathy towards Ilan's loacker hunger.
+
+Examples of unacceptable behavior by participants include:
+
+* Using Sambosak instead of loacker in pull requests.
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. 
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at ilan@testim.ioorg. All
```

Comment:
This email doesn't exist

Reply:
Good catch! Thank you for your commitment towards the lesson02 community and for such a swift review.